servers/src/main/java/tachyon/worker/block/BlockStore.java

```diff
@@ -176,6 +176,13 @@ TempBlockMeta createBlockMeta(long userId, long blockId, BlockStoreLocation loca
   void accessBlock(long userId, long blockId) throws IOException;
 
   /**
+   * Get the block metadata manager.
+   *
+   * @return block metadata manager
+   */
+  BlockMetadataManager getBlockMetadataManager();
```

Comment:
Yeah you can add the two functions in `blockstore` and have the `blockstore` return the result from calling the `blockmetadatamanager`